tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -233,3 +313,73 @@ def test_harvesting_arxiv_workflow_manual_accepted(
         obj = eng.processed_objects[0]
         # It was accepted
         assert obj.status == ObjectStatus.COMPLETED
+
+
+def test_set_document_type_and_refereed_all_proceedings_none_refereed(workflow_app, insert_journals_in_db, workflow):
+    workflow.data = json.loads(pkg_resources.resource_string(
+                __name__, os.path.join('fixtures', 'lit_record_article_all_journals_proceedings_and_none_refereed.json')))
+
+    set_document_type_and_refereed(workflow, None)
+
+    assert 'conference paper' in workflow.data['document_type']
+    assert 'refereed' not in workflow.data
+
+
+def test_set_document_type_and_refereed_all_proceedings_none_refereed_no_effect(workflow_app, insert_journals_in_db, workflow):
+    workflow.data = json.loads(pkg_resources.resource_string(
+                __name__, os.path.join('fixtures', 'lit_record_all_journals_proceedings_and_none_refereed.json')))
+
+    set_document_type_and_refereed(workflow, None)
+
+    assert 'conference paper' not in workflow.data['document_type']
```

Comment:
these tests are really hard to check: one needs to look at this file for assertions, then at the `lit` fixture for the content of the record, then at 2 journal fixtures to check what's in them, so 4 files to understand exactly what is being asserted. May I suggest:
1. Inline the `lit` fixtures: they are not being reused, so it's fine to have the content of each record inside the test.
2. Name the journals in a meaningful way: `Refereed J.`,`Refereed and Proceedings J.`, etc., with some letters (`A`, `B`, etc.) at the end if you need to have more than one of the same type.